src/main/java/org/variantsync/diffdetective/diff/git/GitDiffer.java

```diff
@@ -317,18 +327,48 @@ private static CommitDiffResult getPatchDiffs(
                 diffFormatter.format(diffEntry);
                 final String gitDiff = outputStream.toString(StandardCharsets.UTF_8);
                 final String filename = diffEntry.getOldPath();
+
+                final Matcher matcher = DIFF_HEADER_PATTERN.matcher(gitDiff);
+                final String strippedDiff;
+                if (matcher.find()) {
+                    strippedDiff = gitDiff.substring(matcher.end() + 1);
+                } else {
+                    strippedDiff = gitDiff;
+                }
+
                 try {
-                    final BufferedReader file = getBeforeFullFile(git, parentCommit, filename);
-                    final PatchDiff patchDiff =
-                        createPatchDiff(
+                    final String fullDiff = switch (diffEntry.getChangeType()) {
+                        case ADD, DELETE -> {
+                            // The first lines contains meta information "@@ ... " that we want to skip.
+                            final String[] hunkBeginAndRest = strippedDiff.split(StringUtils.LINEBREAK_REGEX_RAW, 2);
```

Comment:
You can use `StringUtils.LINEBREAK_REGEX.split(strippedDiff, 2)` to avoid introducing `LINEBREAK_REGEX_RAW` and recompiling the regex again.